docs/src/pages/components/tree-view/tree-view.md

```diff
@@ -62,3 +62,7 @@ const data = {
 (WAI-ARIA: https://www.w3.org/TR/wai-aria-practices/#TreeView)
 
 The component follows the WAI-ARIA authoring practices.
+
+### Best Practices
```

Comment:
Since the name is required we should find a stronger wording here. The current wording would make sense if the name was a SHOULD but it's a MUST.